src/renderers/testing/ReactTestRendererFiber.js

```diff
@@ -18,14 +18,15 @@ var ReactGenericBatching = require('ReactGenericBatching');
 var emptyObject = require('emptyObject');
 
 import type { TestRendererOptions } from 'ReactTestMount';
+import type { ReactText } from 'ReactTypes';
 
 type ReactTestRendererJSON = {|
   type : string,
-  props : {[propName: string] : string },
+  props : {[propName: string] : any },
```

Comment:
yeah, I think this is okay. I thought the same thing when grabbing this from the stack implementation. 👍